src/main/java/fi/helsinki/cs/tmc/core/configuration/TmcSettings.java

```diff
@@ -3,6 +3,7 @@
 import fi.helsinki.cs.tmc.core.domain.Course;
 
 import com.google.common.base.Optional;
+import java.nio.file.Path;
```

Comment:
Import order should be:
- All static imports in a single group
- fi.helsinki.cs.tmc imports (only if this source file is in the fi.helsinki.cs.tmc package space)
- Third-party imports, one group per top-level package, in ASCII sort order
- for example: android, com, junit, org, sun
- java imports
- javax imports